pkg/node/node.go

```diff
@@ -158,7 +156,7 @@ const (
 	basePrice   = 10000
 )
 
-func NewBee(addr string, swarmAddress swarm.Address, publicKey ecdsa.PublicKey, signer crypto.Signer, networkID uint64, logger logging.Logger, libp2pPrivateKey, pssPrivateKey *ecdsa.PrivateKey, o Options) (b *Bee, err error) {
+func NewBee(addr string, publicKey *ecdsa.PublicKey, signer crypto.Signer, networkID uint64, logger logging.Logger, libp2pPrivateKey, pssPrivateKey *ecdsa.PrivateKey, o *Options) (b *Bee, err error) {
```

Comment:
why change to pointer semantics?

Reply:
there was some DeepSource (ci checks) complaints about passing in large objects without a pointer.